From 8c25f9cc15d3a2e1123db2b3ba9377530e53ff47 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sun, 30 Apr 2017 11:34:23 +0800
Subject: [PATCH 42/78] arm64: allwinner: a64: add regulators for Pinebook LCD
 SimpleFB

Pinebook's LCD display relies on some regulators.

Add them to the LCD SimpleFB device node.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts
index daf7d8e944c1..4873b9e8fbf4 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinebook.dts
@@ -279,6 +279,12 @@
 	regulator-name = "vcc-rtc";
 };
 
+&simplefb_lcd {
+	vcc-lcd-supply = <&reg_dc1sw>;
+	edp-2v5-supply = <&reg_dldo2>;
+	edp-1v2-supply = <&reg_fldo1>;
+};
+
 &uart0 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&uart0_pins_a>;
-- 
2.13.6

